README.md

```diff
@@ -46,16 +46,15 @@ $ doctest lib/temperature.js
 
 The exit code is 0 if all tests pass, 1 otherwise.
 
-### AMD and CommonJS modules
+### Supported module systems
 
-doctest supports CommonJS modules and partially supports AMD modules:
+doctest supports the following module systems:
```

Comment:
Let's delete this sentence. It now simply restates the heading.

Reply:
:zap: